### PR TITLE
docs: upload json files and instruction to test cases

### PR DESCRIPTION
**Pull request by JuanDavidBuitrago**
## Description

This PR adds json files and instructions to test cases for eox-core, it was made in Postman application.

## Testing instructions

Read 0002-include-test-cases-files.rst

## Additional information

[eox-core test cases](https://docs.google.com/document/d/1gaQyHBGDDkvy_NsFw07MRV6uI2R49CeWCDHzTVi_TiI/edit?usp=sharing)

## Checklist for Merge

- [ ] Tested in a remote environment
- [x] Updated documentation
- [x] Rebased master/main
- [ ] Squashed commits

### CODE REVIEW

**Review by Henrrypg — DISMISSED** [review]

**Review by dcoa — APPROVED** [review]

File: docs/decisions/0002-include-test-cases-files.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+###################
```
Comment (MaferMazu) [inline docs/decisions/0002-include-test-cases-files.rst:1]: I don't think the decision folder is the place for that documentation. 
https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#adrs
Maybe is a good idea to create a folder like help_for_devs or dev_tools or something like that. Later the information will be structured but definitely what is being added is not a decision.

What do you think?

**Review by MaferMazu — DISMISSED** [review]

File: docs/decisions/0002-include-test-cases-files.rst (hunk 1 above)
Comment (JuanDavidBuitrago) [reply docs/decisions/0002-include-test-cases-files.rst]: That seems fine to me. I have created a `help_for_devs` folder where the instructions for using the json files are.

Thanks @MaferMazu for your note!!!

**Comment by MaferMazu** [comment]
Thanks, @JuanDavidBuitrago. I don't know if we can add the end line in the json files. If we can do it without breaking anything, can we add it?

**Comment by JuanDavidBuitrago** [comment]
> Thanks, @JuanDavidBuitrago. I don't know if we can add the end line in the json files. If we can do it without breaking anything, can we add it?

@MaferMazu, I added a new line at the end of the file, there isn't a problem with that.

**Review by Henrrypg — APPROVED** [review]

**Review by dcoa — APPROVED** [review]

**Pull request merged**